src/components/SubMenu.vue

```diff
@@ -1,17 +1,25 @@
 <template>
-  <q-menu auto-close>
+  <q-menu ref="qMenu" :anchor="anchor" :self="self">
     <q-list style="min-width: 100px">
       <q-item
         clickable
         v-for="menu in menuItem.attributes.children.data"
         :key="menu.id"
+        @click="close(menu)"
```

Comment:
`auto-close` is useful only when ALL menu options should close the menu, which is not the current case. Therefore, we need to control who and when can close the menu.